unittests/test_versioning.py

```diff
@@ -17,23 +17,17 @@ def test_version_format(self):
         Version('1.2-dev0')
         Version('1.2-dev5')
         Version('1.2.3-dev2')
-        with pytest.raises(ValueError):
-            Version(None)
+        with pytest.raises(ValueError): Version(None)
 
```

Comment:
Without new lines in between now. That was the whole point of moving everything in a single line.